pontoon/base/models.py

```diff
@@ -1106,6 +1115,11 @@ class Project(AggregatedStats):
         Prevent project from syncing with VCS.
     """)
 
+    system_project = models.BooleanField(default=False, help_text="""
+        System projects are built into Pontoon. They are accessible from the
+        translate view, but hidden from dashboards.
+    """)
+
```

Comment:
Now that you added the `help_text`, you'll also need to update the migration.

If you've already applied it, first unapply it with:
`./manage.py migrate base 0126`

Then delete the `0127_system_project_flag.py` file and create a new migration with:
`./manage.py makemigrations base`